api/devices/models.py

```diff
@@ -13,6 +13,7 @@ class Devices(Base, Utility):
     name = Column(String, nullable=False)
     device_type = Column(String, nullable=False)
     date_added = Column(DateTime, nullable=False)
+    date_updated = Column(DateTime, nullable=False)
```

Comment:
I think you need to set a default value for this field. e.g `date_updated = Column(String, default=datetime.datetime.now)`

Reply:
@vic3king : that presents issues on testing, therefore, I have opted set nullable to true (nullable=True) and my justification for this is that whoever gets tasked to update a specific record  could manipulate nullable to false (nullable=False), set a default value if any, come up with a separate update migration file to handle that scope and also fix the tests.